src/lib/features/feature-toggle/feature-strategies-read-model.ts

```diff
@@ -52,17 +63,17 @@ export class FeatureStrategiesReadModel implements IFeatureStrategiesReadModel {
         environment: string;
         count: number;
     } | null> {
-        const rows = await this.db('feature_strategies')
+        const rows = await this.activeStrategies()
             .select(
                 'feature_name',
                 'environment',
                 this.db.raw(
                     "MAX(coalesce(jsonb_array_length(constraint_value->'values'), 0)) as max_values_count",
                 ),
             )
-            .from(
+            .crossJoin(
                 this.db.raw(
-                    'feature_strategies, jsonb_array_elements(constraints) AS constraint_value',
```

Comment:
this had to be rewritten to cross join after adding join to the feature table so that the syntax is correct